src/shogun/base/SGObject.h

```diff
@@ -735,9 +731,9 @@ class CSGObject
 		BaseTag tag(name);
 		AnyParameterProperties properties(
 			"Dynamic parameter",
-			!AnyParameterProperties::HYPER |
-			!AnyParameterProperties::GRADIENT |
-			!AnyParameterProperties::MODEL);
+			ParameterProperties::HYPER |
+			ParameterProperties::GRADIENT |
+            ParameterProperties::MODEL);
```

Comment:
the whitespaces seem weird here

Reply:
Yup, and should change it to default to `false` instead of all `true`.